website/blog/2024-06-26-llm-as-judge/index.md

```diff
@@ -0,0 +1,522 @@
+---
+title: LLM as judge
+description: Perform LLM Evaluations with custom metrics
+slug: llm-as-judge
+authors: [pedro-azevedo,rahul-pandey]
+tags: [genai,mlflow-evalaute]
+thumbnail: img/blog/llm-as-judge.png
+---
+
+In this blog post, we'll dive on a journey to revolutionize how we evaluate language models. We'll explore the power of MLflow Evaluate and harness the capabilities of Large Language Models (LLMs) as impartial judges. By the end, you'll learn how to create custom metrics, implement LLM-based evaluation, and apply these techniques to real-world scenarios. Get ready to transform your model assessment process and gain deeper insights into your AI's performance!
+
+## The Challenge of Evaluating Language Models
+
+Imagine you're part of a global travel agency, "WorldWide Wandercorp," that's expanding its reach to Spanish-speaking countries. Your team has developed an AI-powered translation system to help create culturally appropriate marketing materials and customer communications. However, as you begin to use this system, you realize that traditional evaluation metrics fall short in capturing the nuances of language translation, especially when it comes to preserving cultural context and idiomatic expressions. You want to understand the materials in which your translation is failing so you can improve the model or manually translate them.
```

Comment:
Suggest expanding this a little bit to note:
- How are "traditional evaluation metrics" falling short? Can you provide any examples of these metrics? this will make it easier for readers to understand the role of LLM-as-judge evaluation.
- A concrete example here would be great. "The system translated this idiom literally as this. The literal translation loses all of this meaning, even though it scores well on [traditional metric]. We need a measurement approach that can ..."